docs/data-tools/index.md

```diff
@@ -117,7 +119,7 @@ Learn how to use data tools in Visual Studio to develop and test data solutions
             <div class="cardPadding">
                 <div class="card">
                     <div class="cardText">
-                        <p><Access data in Visual Studio></p>
+                        <p>Access data in Visual Studio</p>
```

Comment:
The containing angle brackets looked like it was unintended, but the top level disable will also ignore this if you want to undo it